artiq/dashboard/moninj.py

```diff
@@ -398,7 +398,7 @@ async def core_connector(self):
             new_core_connection = CommMonInj(self.monitor_cb, self.injection_status_cb,
                     self.disconnect_cb)
             try:
-                await new_core_connection.connect(self.core_addr, 1383)
+                await new_core_connection.connect("localhost", 2383)
```

Comment:
I swear this wasn't here this morning...
@stevefan1999 maybe refrain from force pushing for a while, it is easy to miss these stuff.

Reply:
this is unintended

Reply:
Ok, so from what I see this line's equivalent is in #1753, i.e. the proxy is not used until #1753. But its title literally "Refactor", which shouldn't justify enabling a feature.

I am raising it here because I am confused why the proxy enabled in a separate PR that intends to refactor only, when it is implemented in this PR.